spanner/leaderboard/README.md

```diff
@@ -0,0 +1,77 @@
+# Java Cloud Spanner Sample Leaderboard Application
+
+A leaderboard sample that uses the Cloud Spanner commit timestamp feature and demonstrates
+how to call the [Google Cloud Spanner API](https://cloud.google.com/spanner/docs/)
+using the [Google Cloud Client Library for Java](https://github.com/GoogleCloudPlatform/google-cloud-java).
+
+This sample requires [Java](https://www.java.com/en/download/) and [Maven](http://maven.apache.org/) for building the application.
+
+This sample includes extra directories `step5`, `step6`, and `step7` that contain partial versions of this sample application. These directories are intended to provide guidance as part of a separate Codelab walk-through where the application is built in the following stages
+that correspond to the steps in Codelab:
+
+* step5 - Create the sample database along with the tables Players and Scores.
+* step6 - Populate the Players and Scores tables with sample data.
+* step7 - Run sample queries including sorting the results by timestamp.
+
+If you only want to run the complete sample refer to the application in the `complete` directory.
+
+
+## Build and Run
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/java/docs/setup).**
+
+2.  Enable APIs for your project.
+    [Click here](https://console.cloud.google.com/flows/enableapi?apiid=spanner.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Google Cloud Spanner API.
+
+3.  Create a Cloud Spanner instance via the Cloud Plaform Console's
+    [Cloud Spanner section](http://console.cloud.google.com/spanner).
+
+4.  In a terminal shell, change directory to the version of the application you want to run:
+    ```
+    cd complete
+    ```
+
+5.  Run the following Maven command to build the application:
+    ```
+    mvn install -DskipTests
```

Comment:
I might have suggested this before, but it might be easier to avoid installing and instead use [mvn to execute directly ](https://www.mojohaus.org/exec-maven-plugin/exec-mojo.html) (up to you)

Reply:
Opting to leave this as is for now.